homeassistant/components/casatunes/media_player.py

```diff
@@ -148,16 +137,14 @@ def shuffle(self):
     @property
     def state(self):
         """Return the state of the device."""
-        status = self._player.status
+        status_states = {
+            0: STATE_IDLE,
+            1: STATE_PAUSED,
+            2: STATE_PLAYING,
+            3: STATE_ON
+        }
 
-        if status == 0:
-            return STATE_IDLE
-        elif status == 1:
-            return STATE_PAUSED
-        elif status == 2:
-            return STATE_PLAYING
-        else:
-            return STATE_ON
+        return status_states.get(self._player.status, STATE_IDLE)
```

Comment:
Is idle the correct default here?  Should it really be unknown?  (Return `none` to indicate `STATE_UNKNOWN`)